subcommand failing when using CPM_SOURCE_CACHE on Windows

When I active the CPM_SOURCE_CACHE, I get a subcommand failed error. It appears to be originating from [this](https://github.com/TheLartians/CPM.cmake/blob/464ba554c23c6aa60443bd9c77005beff82ff810/cmake/CPM.cmake#L221) line.

**executing:**
cmake -B_build -GNinja -DCMAKE_TOOLCHAIN_FILE="../../cmake/arm-gcc-cortex-m4.cmake" -DCPM_SOURCE_CACHE="C:\Users\ryanw/.cpm"

**results in:**
[1/9] Performing download step for 'threadx-populate'
':' is not recognized as an internal or external command,
operable program or batch file.
FAILED: threadx-populate-prefix/src/threadx-populate-stamp/threadx-populate-download
cmd.exe /C "cd /D D:\projects\iot\getting-started\Renesas\AE-CLOUD2\_build\_deps\threadx-subbuild\threadx-populate-prefix\src && : && "C:\Program Files\CMake\bin\cmake.exe" -E touch D:/projects/iot/getting-started/Renesas/AE-CLOUD2/_bu
ild/_deps/threadx-subbuild/threadx-populate-prefix/src/threadx-populate-stamp/threadx-populate-download"
ninja: build stopped: subcommand failed.

--- a/cmake/CPM.cmake
+++ b/cmake/CPM.cmake
@@ -28,7 +28,7 @@
 
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
 
-set(CURRENT_CPM_VERSION 0.19)
+set(CURRENT_CPM_VERSION 0.20)
 
 if(CPM_DIRECTORY)
   if(NOT ${CPM_DIRECTORY} MATCHES ${CMAKE_CURRENT_LIST_DIR})
@@ -218,7 +218,8 @@ function(CPMAddPackage)
     set(download_directory ${CPM_SOURCE_CACHE}/${lower_case_name}/${origin_hash})
     list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS SOURCE_DIR ${download_directory})
     if (EXISTS ${download_directory})
-      list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS DOWNLOAD_COMMAND ";")
+      # disable the download command to allow offline builds
+      list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS DOWNLOAD_COMMAND "${CMAKE_COMMAND};-E;true")
       set(PACKAGE_INFO "${download_directory}")
     else()
       # remove timestamps so CMake will re-download the dependency

--- a/cmake/CPM.cmake
+++ b/cmake/CPM.cmake
@@ -219,7 +219,7 @@ function(CPMAddPackage)
     list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS SOURCE_DIR ${download_directory})
     if (EXISTS ${download_directory})
       # disable the download command to allow offline builds
-      list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS DOWNLOAD_COMMAND "${CMAKE_COMMAND};-E;true")
+      list(APPEND FETCH_CONTENT_DECLARE_EXTRA_OPTS DOWNLOAD_COMMAND "${CMAKE_COMMAND}")
       set(PACKAGE_INFO "${download_directory}")
     else()
       # remove timestamps so CMake will re-download the dependency

